Error when trying to export attendees

Exporting attendees causes errors in the console. Download screen does not open. 

```
state-machine.js:86 Error while exporting TypeError: Cannot read properties of null (reading 'state')
    at attendees.js:59:1
(anonymous) @ state-machine.js:86
attendees.js:70 Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'toUpperCase')
    at attendees.js:70:1
```

Info from Sentry: https://sentry.fossasia.org/organizations/eventyay/issues/4281/

{snip} {console.error("Error while exporting",e),n.notify.error(t.toUpperCase()+" "+n.l10n.t("Export has failed."))}).finally(function(){n.set("isL {snip}

**Download**
[error.odt](https://github.com/fossasia/open-event-frontend/files/12223059/error.odt)

"Task is going on" screen message also stops after a few times.

![Screenshot from 2023-08-01 02-20-49](https://github.com/fossasia/open-event-frontend/assets/1583873/bc6bdae8-3dd1-4e0c-bd25-cbd0af435d2a)


``

Compare https://eventyay.com/events/8f889410/tickets/attendees/completed

--- a/app/components/public/session-item.js
+++ b/app/components/public/session-item.js
@@ -29,7 +29,7 @@ export default class SessionItem extends Component {
   }
 
   @action
-  async setUpComponent(){
+  async setUpComponent() {
     const { speakers } = await this.args.session;
     const sortedSpeakers = {};
     const sessionId = this.args.session.id;

--- a/app/controllers/events/view/tickets/attendees.js
+++ b/app/controllers/events/view/tickets/attendees.js
@@ -32,7 +32,7 @@ export default class AttendeesController extends Controller {
             window.location = exportJobStatus.result.download_url;
             this.notify.success(this.l10n.t('Download Ready'));
           } else if (exportJobStatus.state === 'WAITING') {
-            this.requestLoop(exportJobInfo);
+            this.requestLoop(exportJobInfo, mode);
             this.notify.alert(this.l10n.t('Task is going on.'));
           } else {
             this.notify.error(mode.toUpperCase() + ' ' + this.l10n.t('Export has failed.'));

--- a/app/controllers/events/view/tickets/orders.js
+++ b/app/controllers/events/view/tickets/orders.js
@@ -32,7 +32,7 @@ export default class OrdersController extends Controller {
             window.location = exportJobStatus.result.download_url;
             this.notify.success(this.l10n.t('Download Ready'));
           } else if (exportJobStatus.state === 'WAITING') {
-            this.requestLoop(exportJobInfo);
+            this.requestLoop(exportJobInfo, mode);
             this.notify.alert(this.l10n.t('Task is going on.'));
           } else {
             this.notify.error(mode.toUpperCase() + ' ' + this.l10n.t('Export has failed.'));

